radixdlt/src/main/java/com/radixdlt/mempool/MempoolDuplicateException.java

```diff
@@ -17,14 +17,14 @@
 
 package com.radixdlt.mempool;
 
-import com.radixdlt.middleware2.LedgerAtom;
+import com.radixdlt.middleware2.ClientAtom;
 
 /**
  * Exception thrown when an attempt to add new items would
```

Comment:
Appreciate this isn't new, just realised this description doesn't match the exception name ("capacity" vs "duplicate")

Reply:
Looks like I made copypasta.